po/R-zh_CN.po

```diff
@@ -1940,95 +1940,97 @@ msgid "When x's column ('"
 msgstr ""
 
 msgid "') is character, the corresponding column in y ('"
-msgstr ""
+msgstr "') 是字符，y 中相应的列 ('"
 
 msgid "') should be factor or character, but found incompatible type '"
-msgstr ""
+msgstr "') 应该是因子或字符，然而此类型并不兼容：'"
 
 msgid "') is factor, the corresponding column in y ('"
-msgstr ""
+msgstr "') 是因子，y 中相应的列 ('"
 
 msgid "') should be character or factor, but found incompatible type '"
-msgstr ""
+msgstr "') 应该是字符或因子，然而此类型并不兼容：'"
```

Comment:
probably we should just change the order to `factor or character` and then this will merge with the earlier message...